### PR TITLE
Added tapBackgroundToDismiss to LightBox screens

### PR DESCRIPTION
**Pull request by BrendonSled**
Added param tapBackgroundToDismiss to LightBox screens to allow tapping on background views to dismiss the LightBox screen

### CODE REVIEW

File: android/app/src/main/java/com/reactnativenavigation/views/LightBox.java [hunk 1]
```diff
@@ -54,6 +54,15 @@ private void createContent(final Context context, LightBoxParams params) {
         lightBox.setBackgroundColor(params.backgroundColor.getColor());
         lightBox.addView(content, lp);
 
+        if (params.tapBackgroundToDismiss) {
+            lightBox.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    hide();
```
Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: You better call `destroy()` which dismisses the lightBos (Dialog) and unmounts the react view.

**Comment by guyca** [comment]
Amazing! Please see me comment regarding `hide()`

**Comment by BrendonSled** [comment]
Destroy should be called, hide calls animatedHide and when the animation end destroy is called.

File: android/app/src/main/java/com/reactnativenavigation/views/LightBox.java (hunk 1 above)
Comment (BrendonSled) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: `destroy()` is called after the animation finishes

Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: I don't think so. Once the animation ends, the Dialog is dismissed and the onDismiss listener is called. How is `destroy` getting called?

Comment (BrendonSled) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: Both BottomTabsLayout/SingleScreenLayout both use the `.hide()` method.

If I use the `.destroy()` the dialog will close without an animation

Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: You're right, `destroy` should be called in `onAnimationEnd`.

Comment (BrendonSled) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: ok I've made that change @guyca

Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/views/LightBox.java]: Great!

**Pull request merged**